src/main/java/org/profit/telegrambot/AntiHardcode.java

```diff
@@ -0,0 +1,64 @@
+package org.profit.telegrambot;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.ReplyKeyboardMarkup;
+
+import java.util.ArrayList;
+
+public class AntiHardcode {
+
+   static final String botDescription = """
+            This is a test bot for Java courses disguised as a tea shop.
+
+            It has two simple and one inline keyboards, can send images, and receive them.
+
+            More features coming soon :)""";
+
+   static final String greetingMessage = """
+            Welcome to our Tea Shop!
+
+            We have a wide selection of traditional Chinese tea for reasonable prices.
+
+            Just click 'Go Shopping' to see what we can offer you!
+
+            For any additional information click 'Shop's Info'""";
+
+    protected static final String token = "5793024716:AAHBn08nBwGe0D864bLUFX4UtFhLt412ppk";
```

Comment:
# [ACROSS PR]

Please, talk to Maksym about the code style of your project. Also, in his parallel PR I left a comment regarding the project code style. That one is relevant here as well.